src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -16,8 +16,10 @@ import {
   TouchableOpacity,
   View
 } from 'react-native';
-import {useRoute} from '@react-navigation/core';
+import {useNavigation, useRoute} from '@react-navigation/core';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useNavigation` hook is added to navigate to different screens. This change is necessary for the new feature of adding participants to a group chat.
```diff
- import {useRoute} from '@react-navigation/core';
+ import {useNavigation, useRoute} from '@react-navigation/core';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->